subprojects/docs/build.gradle

```diff
@@ -620,13 +620,18 @@ tasks.named("docsTest") { task ->
         // disable sanityCheck of 'structuring-software-projects' in any case due to deprecation warning in Android project
         excludeTestsMatching "org.gradle.docs.samples.*.structuring-software-projects*_sanityCheck.sample"
 
-        // Do not execute Java Module tests on JVM < 9
-        if (!javaVersion.java9Compatible) {
-            excludeTestsMatching "org.gradle.docs.samples.*.java-modules-*.sample"
-            excludeTestsMatching "org.gradle.docs.samples.*.incubating-java-modules-*.sample"
-            excludeTestsMatching 'org.gradle.docs.samples.*.snippet-java-application*.sample'
-            excludeTestsMatching 'org.gradle.docs.samples.*.snippet-java-library-module*.sample'
-            excludeTestsMatching 'org.gradle.docs.samples.*.snippet-testing-patch-module*.sample'
+        /*
+         * Do not execute tests which include additional compiler warnings that cause output comparisons to fail on JDK 20:
```

Comment:
I think we should resolve this in a separate PR since it is not related to this change. 

Can we split this off into a separate PR in order to ensure we resolve this more quickly?

Reply:
This is just a workaround.  Bo said this was fixed now.  I'll try removing it after rebasing.

Reply:
Not fixed in the latest master.

Reply:
This is removed now, as the latest in master has fixed this problem.